Scripts/Global/UtilityFunctions.gd

```diff
@@ -15,3 +15,10 @@ static func get_all_Children(root_node, stop_class = null) -> Array[Node]: ## Ge
 
 static func in_map(cell: Vector2): ## detects if position is in map (for error handeling)
 	return global.map.local_to_map(cell) in global.map.get_used_cells()
+
+## Note: This method ignores class_name declarations. If this object's script has defined a class_name, the base, built-in class name is returned instead.
+static func get_from_type(children: Array, type_name: String) -> int:
```

Comment:
Don't need this. The only implementation I can see here is redundant, see my comments there.